README.md

```diff
@@ -72,6 +72,9 @@ Join our growing community on [Discord](https://discord.gg/vChUXVf9Em) and [GitH
 
 ## Contributing
 
+> [!NOTE]
+> CodeEdit uses [Tuist](https://tuist.io) to manage Xcode intricacies and streamline the development process. To get started, you'll need to [install Tuist](https://docs.tuist.io/guide/introduction/installation.html)  (we recommend via Mise), and then run `tuist generate` to generate the Xcode project.
```

Comment:
This url leads to a 404 page, do you mean https://docs.tuist.io/guides/quick-start/install-tuist ?